build/pmm-client-docker/Dockerfile-alpine

```diff
@@ -0,0 +1,34 @@
+FROM alpine:latest
+
+ENV MONIT_VERSION=5.25.2
+
+# Compile and install monit
+RUN apk add --update make gcc curl zlib-dev libressl-dev musl-dev \
+    && cd /tmp \
+    && curl -sS https://mmonit.com/monit/dist/monit-${MONIT_VERSION}.tar.gz | gunzip -c - | tar -xf - \
+    && cd monit-${MONIT_VERSION} \
+    && ./configure --without-pam \
+    && make \
+    && make install
+
+COPY pmm-client.tar.gz /tmp/pmm-client.tar.gz
+RUN tar -zxpf /tmp/pmm-client.tar.gz -C /tmp \
+    && cd /tmp/pmm-client-* \
+    && bash ./install
+
+
+FROM alpine:latest
+RUN apk add --update --no-cache bash
+RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
```

Comment:
Can you comment in the source code why this is needed?

Reply:
added